crates/katana/primitives/src/genesis/json.rs

```diff
@@ -72,11 +75,11 @@ pub struct GenesisContractJson {
     pub storage: Option<HashMap<StorageKey, StorageValue>>,
 }
 
-#[derive(Debug, serde::Deserialize)]
+#[derive(Debug, Clone, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct GenesisAccountJson {
-    /// The private key of the account.
-    pub private_key: FieldElement,
```

Comment:
I removed the private key field and replace with public key in the JSON format. The private key is only included in `DevGenesisAccount`.

Reply:
this means accounts with included private key (dev accounts) are not possible to be allocated from the JSON format.